api/util/profilers.py

```diff
@@ -44,7 +44,6 @@ def configure(cls, app: Flask):
         # Don't import if we are not profiling
         from pyinstrument import Profiler
 
-        @app.before_first_request
```

Comment:
I removed this tag because I couldn't see how it was doing anything more than @app.before_request (below: line 48) was doing.